assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -138,9 +139,13 @@ const StoryGridView = ({
             <DetailRow>
               <CardTitle
                 title={story.title}
-                status={story.status}
-                author={users[story.author]?.name}
-                displayDate={story.modified}
+                status={isSavedTemplate ? undefined : story.status}
+                secondaryTitle={
+                  isSavedTemplate
+                    ? __('Google', 'web-stories')
+                    : users[story.author]?.name
+                }
+                displayDate={isSavedTemplate ? undefined : story.modified}
```

Comment:
question: wouldn't displayDate and status be undefined if the saved template doesn't have a modified date or status? Couldn't we just do `displayDate={story?.modified}` and get the same results? I think that our dummy data has a modified date and status just because, but it's not something actually getting used anywhere so we wouldn't need it in our reshape for templates.

Reply:
Oh yes. Good point.